docs/markdown/ios/card/main_component.md

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
I'm not really sold on using `/main_component` in the path. It's also not very intuitive that the when the user clicks on a component name, they see the component page — _unless_ it's expandable, then the side nav section expands but the main view doesn't change. I would think clicking the component name would navigate to the "main_component" content and _also_ expand the side nav section (which could be closed by clicking specifically on the chevron). The extra click to see the actual content is frustrating, and seeing "Main component" in the side nav feels clunky.

What do you think?

Reply:
Thanks for the feedback. It's a bit tricky, but other reviewers haven't brought it up. Do we have to resolve it now? It's the first one, and I rather get more feedback before thinking about how to resolve this. "Card" is the parent, then it has children as either the main component and it's subcomponents. We could be redundant and also call it "Card"? not sure what you mean by this: "It's also not very intuitive that the when the user clicks on a component name, they see the component page — unless it's expandable, then the side nav section expands but the main view doesn't change". Click the component name where?

Reply:
What about:
Card
  Card component
  Subcomponents
    Header.Card